src/components/Button/Button.ts

```diff
@@ -118,11 +93,15 @@ Button.defaultProps = {
   plain: false,
   primary: false,
   secondary: false,
-  size: Button.MEGA,
+  size: MEGA,
   stretch: false,
   target: null
 };
 
+Button.KILO = KILO;
+Button.MEGA = MEGA;
+Button.GIGA = GIGA;
```

Comment:
Do you know how to type static properties on a function? I couldn't find anything when I googled it 🤔